__tests__/index.test.js

```diff
@@ -46,15 +46,16 @@ describe("GET /api/reviews/:review_id", () => {
         const { review } = body;
         expect(review).toEqual(
           expect.objectContaining({
-            review_id: expect.any(Number),
-            title: expect.any(String),
-            review_body: expect.any(String),
-            designer: expect.any(String),
-            review_img_url: expect.any(String),
-            votes: expect.any(Number),
-            category: expect.any(String),
-            owner: expect.any(String),
-            created_at: expect.any(String),
```

Comment:
careful that you don't change code unrelated to your ticket. I'd have created a different branch for this